uxarray/dataset.py

```diff
@@ -31,12 +31,12 @@ def open_dataset(grid_file, *args, **kw):
 
     Examples
     --------
-
     Open grid file only
-    >>> mesh = ux.open_dataset("grid_filename.g")
 
+    >>> mesh = ux.open_dataset("grid_filename.g")
 
     Open grid file along with data
+
     >>> mesh_and_data = ux.open_dataset("grid_filename.g", "grid_filename_vortex.nc")
```

Comment:
What do you think about adding two other examples, something like the following (to reflect the features Philip demonstraets in his example notebook, i.e. opening 1 grid file and multiple data files, as well as 1 grid file and an arary that holds multiple grid files):

```
Open grid file along with multiple data files (two or more)
>>> mesh_and_data_2 = ux.open_dataset("grid_filename.g", "grid_filename_vortex_1.nc", "grid_filename_vortex_2.nc")

Open grid file along with a list of data files
>>> data_files = ["grid_filename_vortex_1.nc", "grid_filename_vortex_2.nc", "grid_filename_vortex_3.nc"]
>>> mesh_and_data = ux.open_dataset("grid_filename.g", *data_files)
```

Reply:
This seems like a beneficial addition since it may not be immediately apartment on how to load in multiple data files

Reply:
Sounds good! I'll add your suggestions